src/test/lib/render_test.ts

```diff
@@ -1046,15 +1046,18 @@ suite('render()', () => {
           assert.isTrue(instance.calledSetter);
         });
 
-    test('does not upgrade elements until after parts are established', () => {
-      render(
-          html`
+    // We need Safari to implement customElements.upgrade before we can enable
+    // this.
+    test.skip(
```

Comment:
If we land https://github.com/Polymer/lit-html/pull/847, we can re-enable regardless of Safari.

Reply:
I'll take a look at that ASAP.